From 5c06dc53560668606b72fa0e673c9eb96948ff39 Mon Sep 17 00:00:00 2001
From: Gianfranco Costamagna <costamagnagianfranco@yahoo.it>
Date: Mon, 4 Jan 2021 15:47:21 +0100
Subject: [PATCH] CMakeLists.txt: fix typo preventing lucene++-contrib library
 symlink from being created correctly

The SONAME/SOVERSION weren't evaluated and set correctly because of the wrong target name.
---
 src/contrib/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/contrib/CMakeLists.txt b/src/contrib/CMakeLists.txt
index 46ed8a24..afeccb4e 100644
--- a/src/contrib/CMakeLists.txt
+++ b/src/contrib/CMakeLists.txt
@@ -77,7 +77,7 @@ endif()
 ####################################
 target_compile_options(lucene++-contrib PRIVATE -DLPP_BUILDING_LIB)
 
-set_target_properties(lucene++
+set_target_properties(lucene++-contrib
   PROPERTIES
     COTIRE_CXX_PREFIX_HEADER_INIT "include/ContribInc.h"
     CXX_VISIBILITY_PRESET hidden
